docs/modules/servers/partials/configure/imap.adoc

```diff
@@ -121,7 +121,7 @@ using HAProxy's proxy protocol.
 | proxyFirst
 | Whether proxy frames should be handled before SSL handshakes. This allows setting either the loadbalancer in TCP mode
 (so transparent for SSL then Proxy frames needs to be handled first) or set up SSL termination between proxy and server
-(more suited for some cloud vendors). Defaults to true (TCP transparent).
+(more suited for some cloud vendors). Defaults to false.
```

Comment:
Should the default configuration code be true instead?

Reply:
Yes this would avoid a breaking change...

Fixups welcome.

Reply:
It's false in the code. One or the other honestly. But faulty doc made me waste a morning trying to find the cause of why imap was not working anymore when deploying a new version with this change

Reply:
We reviewed it too. It is a human mistake. Let's avoid a breaking change IMO.